front/lib/swr.ts

```diff
@@ -443,6 +446,9 @@ export function useDataSourceContentNodes({
       headers: { "Content-Type": "application/json" },
       body,
     };
+    if (!url) {
```

Comment:
It should be handled out of the box by SWR.

Reply:
tried that but it failed, fetcher does not accept a null url

Reply:
As per IRL, in that case we go through fetcher; it's a post so keeping as is :+1: